Documentation/community/roadmap.rst

```diff
@@ -14,35 +14,60 @@ broader community. You'll also find here some pointers on how you can
+++-------------------------------------------------+----------------------------------------------------------+
+| Hubble Observability                             | Stable (:ref:`Roadmap Details<rm-hubble-observability>`) |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Hubble CLI<hubble_cli>`                   | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`hubble_ui`                                | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Prometheus metrics<metrics>`              | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+| Cilium Service Mesh                              | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)  |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gs_ingress`                               | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gs_gateway_api`                           | Beta                                                     |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gs_l7_traffic_management`                 | Beta                                                     |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gs_mutual_authentication`                 | Beta                                                     |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`SPIFFE integration<identity_management>`  | Beta                                                     |
+++-------------------------------------------------+----------------------------------------------------------+
+| `Tetragon`_ Security                             | Beta (:ref:`Roadmap Details<rm-tetragon>`)               |
++--------------------------------------------------+----------------------------------------------------------+
 
 "Stable" means that the feature is in use in production (though advanced
 features may still be in beta or in development).
 
 Release Cadence
 ~~~~~~~~~~~~~~~
 
-We aim to make 2-3 point releases per year of Cilium and its core components
+We aim to make 2 to 3 `point releases`_ per year of Cilium and its core components
```

Comment:
We are rather consistently making 2 releases per year, with patch releases every month.